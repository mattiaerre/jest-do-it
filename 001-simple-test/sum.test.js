const sum = require('./sum');

test('2 + 2 = 4', () => {
  const result = sum(2, 2);

  expect(result).toMatchSnapshot();
});
