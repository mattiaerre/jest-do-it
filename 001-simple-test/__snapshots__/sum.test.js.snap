// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`2 + 2 = 4 1`] = `4`;
